test-case-reporting/src/test_result_record.ts

```diff
@@ -18,24 +18,95 @@ import {
   Build,
   DB,
   Job,
+  PageInfo,
+  QueryFunction,
   TestCase,
   TestRun,
   TestStatus,
   Travis,
 } from 'test-case-reporting';
-import {Database, TIMESTAMP_PRECISION} from './db';
+import {Database} from './db';
 import md5 from 'md5';
 
-const msConversionConstant: number = Math.pow(10, 3 - TIMESTAMP_PRECISION);
-
 function getDateFromTimestamp(timestamp: number): Date {
-  return new Date(timestamp * msConversionConstant);
+  return new Date(timestamp);
```

Comment:
At this point, is this function really adding anything? I think we can just use the constructor :)

Reply:
Woop, you're right.
Removed.